public/employers/ucirvine.yml

```diff
@@ -0,0 +1,49 @@
+name: UC Irvine
+aliases:
+  - UCI
+wiki: University of California, Irvine
+image: ucirvine.svg
+officialWebsite: https://uci.edu/
+industries:
+  - Education
+location:
+  city: Irvine
+  state: CA
+  country: us
+  international: false
+  wiki: Irvine, California
+employeesBefore:
+  type: exactly
+  upperBound: 24714
+  year: 2019
+  yearQuarter: Q4
+summary: UCI has cancelled graduation ceremonies amidst the outbreak early on, switched all classes to online, and have made enormous effort to fight COVID-19
+citations:
+  - summary: UCI have cancelled graduation ceremonies
```

Comment:
As it is, this sounds like a negative -- might be worth mentioning that students will still graduate so it has a positive spin.